src/lib/remark-plugins/rewrite-static-hvd-assets/index.ts

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+/**
+ * This is based on mktg-content-workflow's docs transformers
+ * https://github.com/hashicorp/mktg-content-workflows/blob/main/shared/transforms/rewrite-static-assets.ts
+ *
+ * It rewrites image paths to source from the tutorials repository
+ * via the GitHub CDN, which is provided by the assets API in mtkg-content-workflows
+ */
+
+// types
+import { Plugin } from 'unified'
+import { Image } from 'mdast'
+import { isAbsoluteUrl } from 'next/dist/shared/lib/utils'
+import { visit } from 'unist-util-visit'
+import { HVD_FINAL_IMAGE_ROOT_DIR } from '../../../../scripts/extract-hvd-content'
```

Comment:
hate this, we should have a @root alias...

Reply:
`@scripts` won't work here?